packages/lodestar/src/chain/chain.ts

```diff
@@ -242,6 +242,11 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
   }
 
   private processAttestation = async (latestState: BeaconState, attestation: Attestation, attestationHash: Hash) => {
+    const currentEpoch = computeEpochAtSlot(this.config, latestState.slot);
```

Comment:
I think we need to calculate the epoch in a special way here, using `Date.now() - latestState.genesisTime / SLOTS_PER_EPOCH`, since our latestState.slot may be far behind the 'wall-clock' slot.